javaslang-test/src/main/java/javaslang/test/Arbitrary.java

```diff
@@ -117,7 +163,30 @@
         Objects.requireNonNull(f, "f is null");
         return f.apply(this);
     }
-    
+
+    /**
+     * Generates an arbitrary value from a fixed set of values
+     *
+     * @param values A fixed set of values
+     * @param <U> Type of generator value
+     * @return A new generator
+     */
+    @SafeVarargs
+    @SuppressWarnings("varargs")
+    static <U> Arbitrary<U> fixed(U... values) {
+        return forAll(Gen.choose(values));
+    }
+
+    /**
+     * Generates an arbitrary value from a given generator
+     * @param generator A generator to produce arbitrary values
+     * @param <U> Type of generator value
+     * @return A new generator
+     */
+    static <U> Arbitrary<U> forAll(Gen<U> generator) {
+      return size -> generator;
+    }
```

Comment:
Sure, I'd used `forAll` to follow the usage from `Property.def('').forAll(...)` and since `of()` somewhat implies a fixed ( or more, known ) set of values.  One usage of `fixed` made it nicer when static importing it, tho consistency with the rest of the API is probably better.